src/BuildOnLinux.cmake

```diff
@@ -32,6 +33,20 @@ if(BUILD_SEPARATE_OPS)
     install(TARGETS ${sycl_lib} DESTINATION "${TORCH_INSTALL_LIB_DIR}")
   endforeach()
 elseif(BUILD_SPLIT_KERNEL_LIB OR __INTEL_LLVM_COMPILER LESS 20250001 OR ICX_DATE LESS 20241211)
```

Comment:
Please give an informative comment, like `Working with the compilers which don't support device code compression, we have to split kernels into multiple libraries to meet the bin size limitation.`